CMakeLists.txt

```
@@ -7,6 +7,10 @@ project (fastq_to_fasta VERSION 1.0.0)
 set (CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Specify the directories where to store the built archives, libraries and executables
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
```

This will put intermediate files like `libfastq_to_fasta_lib.a` into the `bin` directory, where otherwise are only executables. I suggest to use either the `lib` directory or create a separate `arch` or `archive` directory.

---

whatever you prefer? In the issue it said 
>  output built archives into a bin directory below the build root
> output built libraries into a lib directory below the build root

but  I also wondered if this was intended

---

Wow, that was fast! I suggest 
```suggestion
set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib")
```

---

Alright :)